src/guide/views/card/CardExample.jsx

```diff
@@ -17,27 +18,22 @@ const CardExample = (props) => {
   rave at close of day;
   Rage, rage against the dying of the light.`;
 
+  const arr = [1, 2, 3];
+
   return (
     <Page title={props.route.name}>
-      <Example title="Card with long text">
-        <Card
-          imageSrc="http://pipsum.com/210x150.jpg"
-          title="This is a title"
-          subtitle="This is a subtitle and this is also a part of the same"
-          description={longDescription}
-          hightlightText="Highlight"
-          tooltipText="Tooltip"
-        />
-      </Example>
-      <Example title="Card without long text">
-        <Card
-          imageSrc="http://pipsum.com/210x150.jpg"
-          title="This is a title"
-          subtitle="This is a subtitle"
-          description="Do not go gentle"
-          hightlightText="Highlight"
-          tooltipText="Tooltip Text and its a long one"
-        />
+      <Example>
```

Comment:
Merging the Card and CardHolder Example together makes sense. But 3 cards is in my opinion not enough to see how the CardHolder behaves. It should have at least 5 or 6 cards.